google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -303,6 +304,105 @@ public void crossInstanceRestoreTest()
+    Instant expireTime = Instant.now().plus(Duration.ofHours(6));
+
+    // Create the backup
+    tableAdmin.createBackup(
+        CreateBackupRequest.of(targetCluster, backupId)
+            .setSourceTableId(testTable.getId())
+            .setExpireTime(expireTime));
+
+    try {
+      CopyBackupRequest req =
+          CopyBackupRequest.of(targetCluster, backupId)
+              .setBackupId(copiedBackupId)
+              .setClusterId(targetCluster)
+              .setExpireTime(expireTime);
+      Backup result = tableAdmin.copyBackup(req);
+      assertWithMessage("Got wrong copied backup id in CopyBackup API")
+          .that(result.getId())
+          .isEqualTo(copiedBackupId);
+      assertWithMessage("Got wrong source backup id in CopyBackup API")
+          .that(result.getSourceBackupId())
+          .isEqualTo(backupId);
+      assertWithMessage("Got wrong expire time in CopyBackup API")
+          .that(result.getExpireTime())
+          .isEqualTo(expireTime);
+      assertWithMessage("Got empty start time in CopyBackup API")
+          .that(result.getStartTime())
+          .isNotNull();
+      assertWithMessage("Got wrong state in CopyBackup API")
+          .that(result.getState())
+          .isAnyOf(Backup.State.CREATING, Backup.State.READY);
+
+    } finally {
+      tableAdmin.deleteBackup(targetCluster, copiedBackupId);
+      tableAdmin.deleteBackup(targetCluster, backupId);
+    }
+  }
+
+  @Test
+  public void crossInstanceCopyBackupTest()
+      throws InterruptedException, IOException, ExecutionException, TimeoutException {
+    String backupId = prefixGenerator.newPrefix();
+    String copiedBackupId = prefixGenerator.newPrefix();
+    Instant expireTime = Instant.now().plus(Duration.ofHours(6));
+
+    // Create the backup
+    tableAdmin.createBackup(
+        CreateBackupRequest.of(targetCluster, backupId)
+            .setSourceTableId(testTable.getId())
+            .setExpireTime(expireTime));
+
+    // Set up a new instance to test cross-instance copy. The backup will be copied here
+    String destInstance = prefixGenerator.newPrefix();
+    String destCluster = prefixGenerator.newPrefix();
+    instanceAdmin.createInstance(
+        CreateInstanceRequest.of(destInstance)
+            .addCluster(destCluster, testEnvRule.env().getSecondaryZone(), 1, StorageType.SSD)
+            .setDisplayName("backups-dest-test-instance")
+            .addLabel("state", "readytodelete")
+            .setType(Type.PRODUCTION));
+
+    try (BigtableTableAdminClient destTableAdmin =
+        testEnvRule.env().getTableAdminClientForInstance(destInstance)) {
+
+      try {
+        CopyBackupRequest req =
+            CopyBackupRequest.of(targetCluster, backupId, testEnvRule.env().getInstanceId())
+                .setBackupId(copiedBackupId)
+                .setClusterId(destCluster)
+                .setExpireTime(expireTime);
+        Backup result = destTableAdmin.copyBackup(req);
```

Comment:
Can we do some assertions for a working/correct case?

Reply:
I believe the assertions are for working/correct cases, the error message is just for when the assertion fails.